code/modules/mob/inventory.dm

```diff
@@ -145,6 +145,10 @@
 //Drops the item in our hand - you can specify an item and a location to drop to
 /mob/proc/drop_item(var/obj/item/to_drop, var/atom/Target)
 	//writepanic("[__FILE__].[__LINE__] ([src.type])([usr ? usr.ckey : ""])  \\/mob/proc/drop_item() called tick#: [world.time]")
+
+	if(incorporeal_move == INCORPOREAL_ETHEREAL) //can't drop items while etheral

```

Comment:
Better solved by the introduction of a flag than this, or by whatever method we fixed dropping while ventcrawling with.